articles/virtual-desktop/deploy-diagnostics.md

```diff
@@ -150,7 +150,7 @@ In order to be able to view the health of VMs, you'll need to enable the Log Ana
 
 To deploy the Azure Resource Management template for the diagnostics tool:
 
-1.  Go to the GitHub Azure RDS-Templates page.
+1.  Go to the WVD Diagnostics section of the GitHub Azure RDS-Templates page: https://github.com/Azure/RDS-Templates/tree/master/wvd-templates/diagnostics-sample/deploy
```

Comment:
Please use "Windows Virtual Desktop" instead of "WVD."

Reply:
Also, please make the link a hyperlink.

`Go to the [Windows Virtual Desktop Diagnostics section](https://github.com/Azure/RDS-Templates/tree/master/wvd-templates/diagnostics-sample/deploy) of the GitHub Azure RDS-Templates page.`